### DISCUSSION TITLE
Disable Title Bar Icon Movement When Navigation Pane Is Toggled

### DISCUSSION THREAD

**Discussion by jaredlhamilton** [lepoco/wpfui#911] [Q&A]
When the navigation pane is toggled, the title bar icon moves left and right. Is it possible to have it stay in the left most position and not move at all?

**Comment by Mithicor** [comment]
Oh Geez I never noticed this till you said something... Now my OCD is going to destroy me.

I will try and figure this one out and get back to you for our sanity sake!


UPDATE:

The margin next to the Title is being changed in the code behind of the [NavigationView.Base.cs](https://github.com/lepoco/wpfui/blob/main/src/Wpf.Ui/Controls/NavigationView/NavigationView.Base.cs) file. The values are at line 64 & 65. Not sure this is something we will be able to alter. If I find anything I will update this, but I am not sure at this point.

**Reply by jaredlhamilton** [reply]
Thank you for the update! I set them both to 35 and that fixed it. Is there a reason it wasn't already set to the same value? Does that break something else?

**Reply by Mithicor** [reply]
I thought it might have to do with "LeftFluent" mode, but it doesn't seem to move the icon... Only when the pane is closed. Perhaps it is just an oversight that was missed?

**Reply by symonxdd** [reply]
Hi! I'm also noticing this issue and it really bothers me, is there a way to change the XAML code in my project itself, or does the file in question need to be changed an recompiled or something along those lines?

**Comment by AncientGrief** [comment]
My fix for this just got merged:
https://github.com/lepoco/wpfui/pull/1209#pullrequestreview-2588422583
